src/classes/Entity/Entity.ts

```diff
@@ -1226,7 +1228,7 @@ error(`'${attrs[i]}' has a default value and cannot be removed`)
     ).trim()
 
     // Merge attribute values
-    ExpressionAttributeValues = Object.assign(values, ExpressionAttributeValues)
+    ExpressionAttributeValues = Object.assign(values, ExpressionAttributeValues, UPDATE_EXPRESSION_ATTRIBUTE_VALUES_DEFAULTS)
```

Comment:
TODO when I get home:
move defaults to the beginning to allow users to override them